### PR TITLE
[bitnami/contour] Add loadBalancerClass options

### PR DESCRIPTION
**Pull request by dshatohin**
### Description of the change

This PR adds `loadBalancerClass` field to the `bitnami/contour` chart.

### Benefits

The `loadBalancerClass` enables to use a load balancer implementation other than the cloud provider default.
For example, when using MetalLB.

### Checklist

- [X] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/). This is *not necessary* when the changes only affect README.md files.
- [X] Variables are documented in the values.yaml and added to the `README.md` using [readme-generator-for-helm](https://github.com/bitnami-labs/readme-generator-for-helm)
- [X] Title of the pull request follows this pattern [bitnami/<name_of_the_chart>] Descriptive title
- [X] All commits signed off and in agreement of [Developer Certificate of Origin (DCO)](https://github.com/bitnami/charts/blob/main/CONTRIBUTING.md#sign-your-work)

### CODE REVIEW

File: bitnami/contour/Chart.yaml [hunk 1]
```diff
@@ -25,4 +25,4 @@ sources:
   - https://github.com/envoyproxy/envoy
   - https://github.com/bitnami/containers/tree/main/bitnami/contour
   - https://projectcontour.io
-version: 9.1.7
+version: 9.1.8
```
Comment (jotamartos) [inline bitnami/contour/Chart.yaml:28]: As you are adding new parameters to the chart, please bump the version to a new minor one (9.2.0)

**Review by jotamartos — CHANGES_REQUESTED** [review]

File: bitnami/contour/Chart.yaml (hunk 1 above)
Comment (jotamartos) [reply bitnami/contour/Chart.yaml]: Thanks!

**Comment by jotamartos** [comment]
Hi @dshatohin,

Please remember you need to sign all the commits in the PR. 

https://github.com/bitnami/charts/blob/main/CONTRIBUTING.md#sign-your-work
https://github.com/bitnami/charts/runs/9271607665

**Comment by dshatohin** [comment]
@jotamartos, thank you for pointing out - I've enabled sign off by default for this repository directory

**Review by jotamartos — APPROVED** [review]

**Pull request merged**